### PR TITLE
chore(repo): Improve READMEs

### PR DESCRIPTION
**Pull request by LekoArts**
## Description

While going through all our SDK READMEs I noticed some inconsistencies, outdated information, and generally speaking too much information that already lives in the docs (as source of truth). By linking to the docs we'll make our lives easier in the future since we don't need to change information in multiple places.

Examples preview: https://github.com/clerk/javascript/blob/readme-updates/packages/nextjs/README.md

Fixes ECO-166

## Checklist

- [ ] `npm test` runs as expected.
- [ ] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -79,7 +79,9 @@ To set up your development environment, please follow these steps:
 
    This ensures that all internal TypeScript types are generated and any dependencies between packages are resolving.
 
-For package specific setup, refer to the `Build` section of the specific package (e.g. `packages/<package_name>/README.md#build`).
+Once you're ready to make changes, run `npm run dev` from the monorepo root.
```
Comment (LekoArts) [inline docs/CONTRIBUTING.md:82]: Since I removed the "Build"  instructions from the READMEs I made this addition

File: packages/astro/README.md [hunk 2]
```diff
@@ -22,164 +21,46 @@
 ·
 [Request a Feature](https://feedback.clerk.com/roadmap)
 ·
-[Ask a Question](https://github.com/clerk/javascript/discussions)
+[Get help](https://clerk.com/contact/support?utm_source=github&utm_medium=clerk_astro)
```
Comment (LekoArts) [inline packages/astro/README.md:24]: We disabled Github Discussions so no use to link to that

File: packages/backend/TODOs.md
Comment (LekoArts) [inline packages/backend/TODOs.md:1]: Stale file, cleanup 🧹

File: packages/chrome-extension/README.md [hunk 3]
```diff
@@ -22,131 +21,116 @@
...
 - Node.js `>=18.17.0` or later
+- An existing Clerk application. [Create your account for free](https://dashboard.clerk.com/sign-up?utm_source=github&utm_medium=clerk_chrome_extension).
+- An existing React app (using [Vite](https://crxjs.dev/vite-plugin/) for example)
 
 ### Installation
 
-```shell
-npm install @clerk/chrome-extension
-```
+1. Add `@clerk/chrome-extension` to your project:
```
Comment (LekoArts) [inline packages/chrome-extension/README.md:40]: The Chrome Extension docs are not live yet, so we keep it for now

File: packages/express/README.md [hunk 4]
```diff
@@ -22,67 +21,163 @@
...
-To build the package locally with the TypeScript compiler, run:
-
-```sh
-npm run build
-```
-
 ## Usage
 
-Retrieve your Backend API key from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) screen in your Clerk dashboard and set it as an environment variable in a `.env` file:
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=api-keys) and inside the **API Keys** section copy the publishable key and secret key.
```
Comment (LekoArts) [inline packages/express/README.md:46]: Express docs are not live yet so we keep them here for now

File: packages/tanstack-start/README.md [hunk 5]
```diff
@@ -6,74 +6,57 @@
...
-npm run build
-```
-
-To build the package in watch mode, run the following:
-
-```sh
-npm run dev
-```
-
 ## Usage
```
Comment (LekoArts) [inline packages/tanstack-start/README.md:48]: Once Tanstack Start docs are live we can also change this

**Review by nikosdouvlis — APPROVED** [review]

**Pull request merged**